test/controllers/api/docs_controller_test.exs

```diff
@@ -8,6 +8,8 @@ defmodule HexWeb.API.DocsControllerTest do
   setup do
     User.create(%{username: "eric", email: "eric@mail.com", password: "eric"}, true)
     |> HexWeb.Repo.insert!
+    File.rm_rf!(Application.fetch_env!(:hex_web, :tmp_dir))
+    File.mkdir!(Application.fetch_env!(:hex_web, :tmp_dir))
```

Comment:
right, so in the case I mentioned I was pushing top-level docs for a pre-release but I wanted to do that only if it's the very first release. Leftover files from previous `test "create release"` or from re-running this this particular test was breaking it. Removing specific file(s) instead of everything will fix it - thanks.